scripts/update-package-json.sh

```diff
@@ -1,6 +1,6 @@
 # expects `$repo`, `$tagPrefix` and `$packages` (array) variables to be defined, see e.g. update-javascript.sh
 
-npm i -g corepack # This repository uses Yarn v3 which requires corepack to be installed
+corepack enable # This repository uses Yarn v3 which requires corepack to be enabled
```

Comment:
Let's add a comment with a brief explanation that this will need to be fixed in the future and why and link to the original PR.

Reply:
Good idea Krystof 👍 
Added a comment with 51ac3556c28d606577b96c84ca5d4667c3aa5e66